### DISCUSSION TITLE
How to use the new model?

### DISCUSSION THREAD

**Discussion by AIhasArrived** [openai/whisper#1772] [Q&A]
I have been using whisper, by using this command:
whisper "test.mkv" --language en --model base.en --device cuda --task transcribe

What do I have to do to use the new models? Does it work with english? step by step:
1) download the large model (I saw many other files being from "yetsreday" so I am confused maybe there are other things I should update?
2) then put it inside.. I actually have no idea where are located my models? (I have a "whisper" folder but it has no models (wonder where they went) ?
3) modify the code: 
whisper "test.mkv" --language en --model largev3.en --device cuda --task transcribe
??
Did I miss anything else?

**Comment by glangford** [comment]
There is a typo in your step 3, the model name is incorrect.

You need to be running the 20231106 release,
> and load the model using the name "large-v3". The name "large" now aliases to the latest model in the series, "large-v3".

See this post.

* https://github.com/openai/whisper/discussions/1762

**Reply by LindaQian2828** [reply]
How do I install on Mac?
